<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BlackBerrySettingsWidget</class>
 <widget class="QWidget" name="BlackBerrySettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>50</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QPushButton" name="setupButton">
      <property name="text">
       <string>Setup</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="clearButton">
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>651</width>
     <height>371</height>
    </rect>
   </property>
   <property name="title">
    <string>BlackBerry Settings</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>631</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="SDKPathLabel">
         <property name="text">
          <string>BlackBerry Native SDK</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Utils::PathChooser" name="sdkPath" native="true">
         <zorder>ndkInfosTableView</zorder>
         <zorder>label</zorder>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableView" name="ndkInfosTableView"/>
     </item>
    </layout>
   </widget>
  </widget>
  <zorder>SDKPathLabel</zorder>
  <zorder>verticalLayoutWidget_2</zorder>
  <zorder>groupBox</zorder>
  <zorder>clearButton</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::PathChooser</class>
   <extends>QWidget</extends>
   <header location="global">utils/pathchooser.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
